p384/Cargo.toml

```
@@ -26,15 +26,25 @@ primeorder = { version = "0.13.1", optional = true, path = "../primeorder" }
 serdect = { version = "0.2", optional = true, default-features = false }
 sha2 = { version = "0.10", optional = true, default-features = false }
 
+[target.'cfg(all(target_os = "zkvm", target_arch = "riscv32"))'.dependencies]
+bytemuck = "1"
+# https://github.com/risc0/risc0/pull/2783
+risc0-bigint2 = { git = "https://github.com/risc0/risc0.git", rev = "bdc2ec71ac7104bd44ead8f609962bf53aa68441", features = ["unstable"]}
```

This PR has landed in `risc0-bigint2` version `1.4.0`, so you should be able to switch this to an actual release
```suggestion
risc0-bigint2 = { version = "1.4.0", features = ["unstable"]}
```